crypto/store/loader_file.c

```diff
@@ -764,20 +764,22 @@ static OSSL_STORE_LOADER_CTX *file_open(const OSSL_STORE_LOADER *loader,
         }
     } else {
         BIO *buff = NULL;
-        char peekbuf[4096];
+        char *peekbuf = OPENSSL_malloc(4096);
```

Comment:
Why?  Is that really necessary?  Is this about maximum stack size getting broken?